python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -246,6 +244,19 @@ class LogisticRegression(Base,
 
         loss = "sigmoid"
 
+        if class_weight is not None:
+            if class_weight == 'balanced':
```

Comment:
It would be better to make the parameter default to `balanced` in the init call

Reply:
Scikit-learn uses `None` as default. Don't you think that it might be disturbing to users if  the defaults are different?

Reply:
For some reason I thought `balanced` was the default in Scikit, but I was confusing estimators